components/usage/pkg/db/types.go

```diff
@@ -80,9 +79,21 @@ func (n VarcharTime) IsSet() bool {
 
 // Value implements the driver Valuer interface.
 func (n VarcharTime) Value() (driver.Value, error) {
```

Comment:
This is what will be inserted into SQL statements. Here we use the ISO8601 format to be consistent with existing records.